packages/grep/grep.spec

```
@@ -0,0 +1,33 @@
+Name: %{_cross_os}grep
+Version: 3.6
+Release: 1%{?dist}
+Summary: GNU grep utility
+URL: https://www.gnu.org/software/grep/
+License: GPLv3+
```

We use SPDX license IDs:
```suggestion
License: GPL-3.0-or-later
```